roles/bwc_repos/tasks/bwc_repos_apt.yml

```diff
@@ -0,0 +1,40 @@
+---
+
+- name: Assert that master_token is specified
+  fail: msg="License key must be supplied for BWC enterprise installation."
+  when: license is not defined
```

Comment:
Please include all possible vars used in the roles in `defaults.yml` with comments and decent README.
With that, instead of relying on `is not defined` we can rely on is empty conditional check.